### PR TITLE
Updated Error messages on Mandatory fields of OIDC Scopes

### PR DESCRIPTION
**Pull request by divyaamunugama**
### Purpose
Updated Error messages on OIDC Scopes

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on final implementation.
- [ ] Documentation provided. (Add links if there's any)
- [ ] Unit tests provided. (Add links if there's any)
- [ ] Integration tests provided. (Add links if there's any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

**Review by yasinmiran — APPROVED** [review]

File: apps/console/src/features/oidc-scopes/components/wizards/add-oidc-scope-form.tsx [hunk 1]
```diff
@@ -85,10 +85,10 @@ let triggerPreviousForm: () => void;
             validate={ (values): any => {
                 const errors:any = {};
                 if (!values.scopeName && !initialValues?.scopeName) {
-                    errors.scopeName="Required";
+                    errors.scopeName="This field cannot be empty";
```
Comment (brionmario) [inline apps/console/src/features/oidc-scopes/components/wizards/add-oidc-scope-form.tsx:88]: Shall we add `i18n` strings here.

**Ref** - https://github.com/wso2/identity-apps/blob/master/modules/i18n/src/translations/en-US/portals/console.ts#L2905

File: modules/i18n/src/translations/en-US/portals/console.ts [hunk 2]
```diff
@@ -2900,7 +2900,7 @@ export const console: ConsoleNS = {
                                 validations: {
                                     invalid: "Email OTP expiry time should be an integer.",
                                     range: "Email OTP expiry time should be between 1 second & 86400 seconds(1 day).",
-                                    required: "Email OTP expiry time is a required field."
+                                    required: "This field cannot be empty"
```
Comment (brionmario) [inline modules/i18n/src/translations/en-US/portals/console.ts:2903]:
```suggestion
                                    required: "Email OTP expiry time is a required field."
```

**Review by inthirakumaaran — APPROVED** [review]

File: apps/console/src/features/oidc-scopes/components/wizards/add-oidc-scope-form.tsx [hunk 3]
```diff
@@ -85,10 +85,12 @@ let triggerPreviousForm: () => void;
             validate={ (values): any => {
                 const errors:any = {};
                 if (!values.scopeName && !initialValues?.scopeName) {
-                    errors.scopeName="Required";
+                    errors.scopeName=t("console:manage.features.oidcScopes.forms.addScopeForm.inputs." +
```
Comment (brionmario) [inline apps/console/src/features/oidc-scopes/components/wizards/add-oidc-scope-form.tsx:88]:
```suggestion
                    errors.scopeName = t("console:manage.features.oidcScopes.forms.addScopeForm.inputs." +
```

File: apps/console/src/features/oidc-scopes/components/wizards/add-oidc-scope-form.tsx [hunk 4]
```diff
@@ -85,10 +85,12 @@ let triggerPreviousForm: () => void;
...
+                    "scopeName.validations.empty");
                 }
                 if (!values.displayName && !initialValues?.displayName) {
-                    errors.displayName="Required";
+                    errors.displayName=t("console:manage.features.oidcScopes.forms.addScopeForm.inputs." +
```
Comment (brionmario) [inline apps/console/src/features/oidc-scopes/components/wizards/add-oidc-scope-form.tsx:92]:
```suggestion
                    errors.displayName = t("console:manage.features.oidcScopes.forms.addScopeForm.inputs." +
```

**Pull request merged**